### ISSUE TITLE
[OJS] Counts dont update when deleting submissions

### ISSUE THREAD

**Issue by alexdryden** [pkp/pkp-lib#6273]
OJS version: 3.2.1-1

Issue: On deleting a submission, the count of submissions displayed for that tab does not update. 

More details: When moving to the next page within a tab the total in the active tab updates but totals in other tabs do not. 

Steps to reproduce:

1. Open a journal with some duplicate submissions that need to be deleted and have not yet been assigned to an editor.
2. Have enough unpublished submissions that there is more than one page of results for the All Active tab
3. Delete one of the one of the submissions from All Active
4. Counts do not change
5. Move to the second page of All Active
6. Count for All Active updates, but Count for Unassigned does not

**Comment by asmecher** [comment]
@NateWr, presumably this is made possible by the rewrite to page construction in the master branch back-end. Is there a good pattern in the codebase to follow?

**Comment by NateWr** [comment]
Yes, there will be some syncing issues here. When a submission is deleted, each list searches its own submissions and updates accordingly if it is part of the items in the list. See [this code](https://github.com/pkp/ui-library/blob/master/src/components/ListPanel/submissions/SubmissionsListPanel.vue#L302-L316).

However, if a submission is part of the total count (`itemsMax`) but not part of the current page of items in the list, the list does not know whether the deleted submission should effect its own total count.

The only way to to fix this is to have each submission list request an update to its own items whenever an item is deleted. This can be a pretty heavy request in large systems and that means we may run into some UX oddities with the time it takes for requests to return. Item totals may update so late that people don't understand why the number has changed.

I think we can move forward with this fix anyway. One option to optimise this would be to add a `/submissionCounts` API endpoint that would mimic the params of the `/submissions` endpoint but only return total counts for the request.
